packages/legend-graph/src/graph/MetaModelConst.ts

```diff
@@ -30,6 +30,7 @@ export const PARSER_SECTION_MARKER = '###';
 export const FUNCTION_SIGNATURE_MULTIPLICITY_INFINITE_TOKEN = 'MANY';
 export const DIRECTORY_PATH_DELIMITER = '/';
 export const SOURCE_INFORMATION_PROPERTY_KEY_SUFFIX = 'sourceInformation';
+export const MINUS_STRING = 'minus';
```

Comment:
this is handled in query-builder, there's a list of supported functions over there, please move it there accordingly, and it should be called `QUERY_BUILDER_SUPPORTED_FUNCTIONS.MINUS` instead